internal/cmd/stack/list.go

```diff
@@ -23,57 +20,40 @@ func listStacks() cli.ActionFunc {
 		case cmd.OutputFormatTable:
 			return listStacksTable(cliCtx)
 		case cmd.OutputFormatJSON:
-			return listStacksJSON(cliCtx.Context)
+			return listStacksJSON(cliCtx)
 		}
 
 		return fmt.Errorf("unknown output format: %v", outputFormat)
 	}
 }
 
-func listStacksJSON(ctx context.Context) error {
-	var query struct {
-		Stacks []stack `graphql:"stacks" json:"stacks,omitempty"`
+func listStacksJSON(ctx *cli.Context) error {
+	stacks, err := searchStacks(ctx.Context, structs.SearchInput{})
```

Comment:
The order here is not the same as previously. Does that matter?

Reply:
Probably no.